test/pom.xml

```diff
@@ -194,5 +194,23 @@
 				</dependency>
 		 </dependencies>
 		</profile>
+		<profile>
+			<id>jdk11</id>
+			<activation>
+				<jdk>11</jdk>
+			</activation>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.apache.maven.plugins</groupId>
+						<artifactId>maven-compiler-plugin</artifactId>
+						<version>3.8.1</version>
+						<configuration>
+							<release>11</release>
+						</configuration>
+					</plugin>
+				</plugins>
+			</build>
+		</profile>
```

Comment:
I would expect this in the parent pom, and other poms to inherit it from there. Does that not work?